imports/ui/components/navigation.js

```diff
@@ -28,6 +41,12 @@ function handleToggleMob() {
     $('#menubar').data('expanded', true);
     $('#menubar').triggerHandler('enter');
   }
+
+  if(menuItemClicked === true) {
```

Comment:
if you look right, the statement "menuItemClicked === true" shows if there's a class named "menubar-pin" on body. I'm actually using the same var as used before to check if you're hovering the menubar.

https://github.com/caike08/BancoTalentosTCC/blob/32da17e3940fcb49083ebd8e7577dce8eb22ae52/imports/ui/components/navigation.js#L33

I just need "click #base" to be true if the class is being used. Otherwise it won't be firing all the time.

However, "click #logout" was a way to handle the "menubar-pin" class avoiding it to remain even if you're no longer on our main panel.